config_example.yaml

```diff
@@ -122,6 +127,7 @@ webhook:
   # customHeaders: # Custom headers to add in POST, useful for Authentication
   #   key: value
   # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
+  # mutualtls: false # if true, checkcert flag will be ignored (server cert will always be checked)
```

Comment:
Are you sure for AWS Outputs? As they already use an authentication system with IAM.

Reply:
Sorry but I don't understand this comment (AWS output is not included in the Mutual TLS support). Could you elaborate a little more?

Reply:
The communication with AWS Services is already with SSL and protected by authentication. What could be the use case of mTLS? I'm not even sure it's used.

Reply:
I've never assumed webhook output was just for AWS services, I thought it was like a general webhook output, which may support (or not) mutual TLS authN without authN headers.

Reply:
I got it, the lines are collapsed and my theme makes colors so closed I didn't noticed. Sorry. 😓